nbs/core.ipynb

```diff
@@ -179,6 +179,9 @@
     "        #remove prediction intervals\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #67.                raise Exception('Summing matrix <code>S</code> and <code>Y_hat_df</code> do not have the same time series, please check.')
</pre><p>Maybe somethinh like: "...do not have the same number of time series..." would make it clearer. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>